src/dmap_import/util_aws.py

```diff
@@ -0,0 +1,46 @@
+import os
+import boto3
+
+from dmap_import.util_logging import ProcessLogger
+
+
+def running_in_aws() -> bool:
+    """
+    return True if running on aws, else False
+    """
+    return bool(os.getenv("AWS_DEFAULT_REGION"))
+
+
+def check_for_parallel_tasks() -> None:
+    """
+    Check that that this task is not already running on ECS
+    """
+    if not running_in_aws():
+        return
+
+    process_logger = ProcessLogger("check_for_tasks")
+    process_logger.log_start()
+
+    client = boto3.client("ecs")
+    dmap_ecs_cluster = "dmap-import"
+    environment_name = os.environ["ENVIRONMENT"]
```

Comment:
suggestion: rather than passing in the ENVIRONMENT and using that to build the task name, I'd pass in the ECS_CLUSTER and ECS_TASK_GROUP and do the calculations on the Terraform side. (Generally speaking, we prefer to pass in the values to the application, rather than the environment name and having the application calculate the values for each environment)

Reply:
This is probably more what I was getting at with my comment, maybe the function implementation could be a lot more general with these available.